src/components/body/BodyCell.ts

```diff
@@ -24,22 +24,21 @@ import { TemplateWrapper } from '../../directives/TemplateWrapper';
     '[style.width]': 'column.width + "px"',
     '[style.height]': 'column.height + "px"'
   },
-  directives: [ TemplateWrapper ]
+  directives: [ TemplateWrapper ],
+  changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class DataTableBodyCell {
 
   @Input() column: TableColumn;
   @Input() row: any;
 
+  @HostBinding('class.datatable-body-cell') isBodySell: boolean = true;
```

Comment:
it should be fixed here https://github.com/swimlane/angular2-data-table/pull/40/commits/a286f3683ba76f941a467fa845e538b01b1e87d1